src/assets/CoffeeScriptAsset.js

```diff
@@ -0,0 +1,22 @@
+const JSAsset = require('./JSAsset');
+const config = require('../utils/config');
+const localRequire = require('../utils/localRequire');
+
+class CoffeeScriptAsset extends JSAsset {
+  async transform() {
```

Comment:
This shouldn't be necessary, after the TypeScript PR this has been fixed by @devongovett in a better way to parse every non js file automatically. This would also break Uglify support.

Reply:
I tried removing the async transform block, but then it stopped working. I'm not sure why it works for TS but not for this.

Reply:
It should look like this:
```Javascript
class CoffeeScriptAsset extends JSAsset {
  async parse(code) {
    // require coffeescript, installed locally in the app
    let coffee = localRequire('coffeescript', this.name);

    // Transpile Module using CoffeeScript and parse result as ast format through babylon
    this.contents = coffee.compile(code, {});
    return await super.parse(this.contents);
  }
}
```